ruleset/testing/tests/gcp.ini

```diff
@@ -0,0 +1,227 @@
+
+rule = 65052
+alert = 3
+decoder = json
+
+; [defense evasion gcp_firewall_rule created]
+; log 1 pass = 
+
+; rule = 65053
+; alert = 3
+; decoder = json
+
+; [defense evasion gcp_firewall_rule deleted]
+; log 1 pass = 
+
+; rule = 65054
+; alert = 3
+; decoder = json
+
+[defense evasion gcp_firewall_rule modified]
+log 1 pass = {"gcp":{"severity":"NOTICE","logName":"projects/wazuh-dev-258815/logs/cloudaudit.googleapis.com%2Factivity","resource":{"type":"gce_firewall_rule","labels":{"project_id":"wazuh-dev-258815","firewall_rule_id":"6352627401320071455"}},"protoPayload":{"requestMetadata":{"callerSuppliedUserAgent":"Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.212 Safari/537.36,gzip(gfe),gzip(gfe)","callerIp":"213.194.154.232"},"request":{"@type":"type.googleapis.com/compute.firewalls.patch"},"authenticationInfo":{"principalEmail":"carlos.ridao@wazuh.com"},"@type":"type.googleapis.com/google.cloud.audit.AuditLog","resourceOriginalState":{"logConfig":{"metadata":"INCLUDE_ALL_METADATA","enable":"false"},"@type":"compute.googleapis.com/patch.state","description":"Allow internal traffic on the default network","priority":"65534","network":"https://www.googleapis.com/compute/v1/projects/wazuh-dev-258815/global/networks/default","selfLink":"https://www.googleapis.com/compute/v1/projects/wazuh-dev-258815/global/firewalls/default-allow-internal","sourceRanges":["10.128.0.0/9"],"selfLinkWithId":"https://www.googleapis.com/compute/v1/projects/wazuh-dev-258815/global/firewalls/6352627401320071455","creationTimestamp":"2021-05-31T12:58:40.887-07:00","name":"default-allow-internal","alloweds":[{"IPProtocol":"tcp","ports":["0-65535"]},{"IPProtocol":"udp","ports":["0-65535"]},{"IPProtocol":"icmp"}],"disabled":"false","id":"6352627401320071455","enableLogging":"false","direction":"INGRESS"},"methodName":"v1.compute.firewalls.patch","resourceName":"projects/wazuh-dev-258815/global/firewalls/default-allow-internal","serviceName":"compute.googleapis.com"},"receiveTimestamp":"2021-06-04T13:26:50.156887429Z","operation":{"last":"true","producer":"compute.googleapis.com","id":"operation-1622813206787-5c3f0a4ba3417-3838b401-5fcfe6fd"},"insertId":"qx0fpcdfo5k","timestamp":"2021-06-04T13:26:49.79784Z"},"integration":"gcp"}
+
+rule = 65055
+alert = 3
+decoder = json
+
+; [defense evasion gcp_logging_bucket deletion]
```

Comment:
Why all those rules commented? If it's needed to create a UT for them do it, otherwise remove it.
I think that, in order to approve this (and all rules PR) all of them should be covered by at least 1 UT.